rancher2/schema_node_template_otc.go

```diff
@@ -0,0 +1,223 @@
+package rancher2
```

Comment:
probably this template should be in another pr?

Reply:
Unfortunately, I need a driver in this file. I can comment all stuff instead of the constant.